### PR TITLE
Update yarn scripts to automate build

### PR DESCRIPTION
**Pull request by rkuykendall**

### CODE REVIEW

File: src/containers.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export interface IDisabledContainerProps {
   className?: any;
 }
 
-export interface IGuardedContainerProps {
+ export interface IGuardedContainerProps  {
```
Comment (kbouchard) [inline src/containers.tsx]: two extra spaces here I think.

File: package.json [hunk 2]
```diff
@@ -10,22 +10,23 @@
     "access": "public"
   },
   "scripts": {
-    "clean": "rm -r dist/*",
-    "build:types": "yarn tsc --emitDeclarationOnly",
+    "build": "npm run build:clean && npm run build:js",
+    "build:clean": "rm -r dist/* || true",
     "build:js": "rollup -c",
-    "build": "npm run build:types && npm run build:js",
+    "build:types": "yarn tsc --emitDeclarationOnly",
```
Comment (kbouchard) [inline package.json]: You're using `npm` everywhere except here, maybe we should get rid off all the yarn stuff since it's not required.

Comment (rkuykendall) [reply package.json]: I was actually unable to get this running in npm for reasons I'm unsure of :/ Agree though that it should be resolved. Maybe you can assert it works for you at some point?

File: src/containers.tsx (hunk 1 above)
Comment (rkuykendall) [reply src/containers.tsx]: 🤦‍♂️ Must have been from when I was playing with lint checks.

**Pull request merged**